alpenhorn/archive.py

```diff
@@ -30,11 +30,14 @@ class ArchiveFileCopy(base_model):
         - 'M': maybe, can delete if we need space
         - 'N': no, attempt to delete
         In all cases we try to keep at least two copies of the file around.
+    size_b : integer
+        Size of file in bytes.
```

Comment:
You should point out that this is the _actual_ size, not the _apparent_ size.  Maybe:
```
Actual size of file on disk in bytes.
```
or
```
Allocated size of file on disk in bytes.
```
Is it worth point out that this will be a multiple of 512?

Reply:
Agreed on both. This is calculated using `os.stat.st_blocks`, which the doc says is the "number of 512-byte blocks allocated for file", so our description could be 
```
allocated size of file on disk in bytes (calculated as a multiple of 512-byte blocks)
```
Sounds OK?

Reply:
Updated as above.